core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20210129_add_entity_statistics_cache_tables.xml

```diff
@@ -0,0 +1,61 @@
+            <column name="WORKSPACE_ID" type="BINARY(16)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ENTITY_TYPE" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ENTITY_TYPE_COUNT" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="create_WORKSPACE_ENTITY_TYPE_STATISTICS_FK">
+        <addForeignKeyConstraint baseColumnNames="WORKSPACE_ID" baseTableName="WORKSPACE_ENTITY_TYPE_STATISTICS" constraintName="FK_WETS_WORKSPACE" deferrable="false" initiallyDeferred="false" onDelete="CASCADE" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="WORKSPACE"/>
+    </changeSet>
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="create_WORKSPACE_ENTITY_TYPE_STATISTICS_INDEX">
+        <createIndex indexName="IDX_WORKSPACE_ENTITY_TYPE_STATS" tableName="WORKSPACE_ENTITY_TYPE_STATISTICS" unique="false">
+            <column name="WORKSPACE_ID"/>
+        </createIndex>
+    </changeSet>
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="create_WORKSPACE_ENTITY_ATTRIBUTE_STATISTICS">
+        <createTable tableName="WORKSPACE_ENTITY_ATTRIBUTE_STATISTICS">
+            <column name="WORKSPACE_ID" type="BINARY(16)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ENTITY_TYPE" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ATTRIBUTE_NAMESPACE" type="VARCHAR(32)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ATTRIBUTE_NAME" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="create_WORKSPACE_ENTITY_ATTRIBUTE_STATISTICS_FK">
+        <addForeignKeyConstraint baseColumnNames="WORKSPACE_ID" baseTableName="WORKSPACE_ENTITY_ATTRIBUTE_STATISTICS" constraintName="FK_WEAS_WORKSPACE" deferrable="false" initiallyDeferred="false" onDelete="CASCADE" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="WORKSPACE"/>
+    </changeSet>
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="create_WORKSPACE_ENTITY_ATTRIBUTE_STATISTICS_INDEX">
+        <createIndex indexName="IDX_WORKSPACE_ENTITY_ATTRIBUTE_STATS" tableName="WORKSPACE_ENTITY_ATTRIBUTE_STATISTICS" unique="false">
+            <column name="WORKSPACE_ID"/>
+        </createIndex>
+    </changeSet>
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="add-cache-updated-ws">
+        <addColumn tableName="WORKSPACE">
+            <column name="entity_cache_last_updated" type="TIMESTAMP(6)" defaultValueDate="1970-01-01 00:00:02"/>
```

Comment:
`1970-01-01 00:00:01` is the earliest timestamp that MySQL supports, so we'll use that one to "ignore" workspaces.

hence why `1970-01-01 00:00:02` is the default

Reply:
golden opportunity here for an easter egg ... somebody's birthday? Some other important date?

Reply:
I am failing to be creative right now :'(